tests/tests/listable.rs

```diff
@@ -223,6 +225,9 @@ macro_rules! test_primitive {
                         test_iter(IntoIterator::into_iter(&slice), &self.0);
                         test_iter(IntoIterator::into_iter(slice), &self.0);
                     }
+
+                    fn visit_value(&mut self, _: Value<'_>) {
+                    }
```

Comment:
nit: weird formatting here, did rustfmt do this? i'd expect it to be
```suggestion
                    fn visit_value(&mut self, _: Value<'_>) {}
```

Reply:
rustfmt runs in CI :shrug: